tradeexecutor/state/portfolio.py

```diff
@@ -217,6 +217,20 @@ def get_open_position_for_pair(self, pair: TradingPairIdentifier) -> Optional[Tr
             if pos.pair.get_identifier() == pair.get_identifier():
                 return pos
         return None
+    
+    def get_closed_position_for_pair(
```

Comment:
`@0xh25 `get_closed_position_for_pair()` -> probably rename to `get_closed_positions_for_pair()` because it is likely there are multiple`